test/lib/ufe/testDeleteCmd.py

```diff
@@ -209,10 +263,14 @@ def testDeleteArgs(self):
         self.assertNotIn(sphereShapeName, sphereChildrenNames)
         self.assertNotIn(ball35Name, propsChildrenNames)
         self.assertNotIn(ball34Name, propsChildrenNames)
-        self.assertFalse(cmds.ls("|pSphere1|pSphereShape1"))
+        self.assertFalse(cmds.objExists("|pSphere1|pSphereShape1"))
```

Comment:
The ls command is actually undoable. So this test wasn't really testing the undo of the cmds.delete from line 250. My new test case caught this since the notifs weren't right.

Reply:
Funny thing is that objExists says its also undoable (in docs), but this case works.

Reply:
Nice catch!  Uh, objExists is undoable?  What does that mean?  It's a pure query...  Maybe if you undo objExists and it returned true, the object will become an anti-matter version of itself?  I'm struggling with this, I admit.